Integrations/integration-ProtectWise.yml

```diff
@@ -203,11 +210,15 @@ script:
                 params.threatCategory, null, null, null, null, null, null, null, token);
             try {
                 var res = JSON.parse(data).events;
+                res.reverse();  # events are fetched in descending order so reversing to process older first
             } catch (ex) {
                 throw 'Error parsing event fetch - ' + data + ' - ' + ex;
             }
             var incidents = [];
             for (var i = 0; i < res.length; i++) {
+                if (incidents.length >= Math.min(parseInt(params.maxFetch), 50)) {
```

Comment:
why are we doing `min` here? if the user sets the max fetch to `51`, then it should so

Reply:
nope, 50 is a hard limit as done with other integrations until there's a platform solution to handle it properly

Reply:
not sure i understand this, but anyway if we want to limit to 50, it should be clear in the parameter display name that max is 50 in anyway

Reply:
https://github.com/demisto/etc/issues/19251

Reply:
im aware of that, but thats not my point
the number `50` is not clear to me, but it doesnt really matter, it should be clear that this is the max so we wont get a bug that someone sets the max to 51 and wont understand why it fetches only 50

Reply:
yup, waiting for writers to approve text